### PR TITLE
release notes for v1.2.2 patch release

### PR DESCRIPTION
**Pull request by mattcary**
/kind feature

```release-note
None
```

/assign @saikat-royc
/assign @cpanato

### CODE REVIEW

File: CHANGELOG/CHANGELOG-1.2.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
-# v1.2.1 - Changelog since v1.2.0
+# v1.2.2 - Changelog since v1.2.1
+
+## Changes by Kind
+
+### Feature
+
+- Update base image to buster-1.5.0. ([#755](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/755), [@mattcary](https://github.com/mattcary))
```
Comment (cpanato) [inline CHANGELOG/CHANGELOG-1.2.md:7]: we now are using buster-1.6.0 -> https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/release-1.2/Dockerfile#L24
this PR(https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/760) update that but did not have a release notes :(

Comment (cpanato) [reply CHANGELOG/CHANGELOG-1.2.md:7]: updated the PR to add the release notes

Comment (mattcary) [reply CHANGELOG/CHANGELOG-1.2.md:7]: udpated

**Review by cpanato — APPROVED** [review]
/lgtm

**Comment by saikat-royc** [comment]
@mattcary  can you please wait for https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/758, I have rebased it, and the presubmits are running

**Comment by mattcary** [comment]
Yup, after that merges I'll update the release notes again.

**Pull request merged**